scripts/publish_test_layer.sh

```diff
@@ -86,4 +86,7 @@ echo "Starting publishing layer for region $REGION..."
 publish_layer $REGION "$aws_version_key"
 rm $LAYER_ZIP
 
+# Open a PR to the documentation repo to automatically bump layer version
```

Comment:
I imagine we explicitly don't want to bump the version when we publish a test layer